src/content-helper/common/settings/types/sidebar-settings.d.ts

```diff
@@ -41,4 +41,5 @@ export interface TitleSuggestionsSettings {
 	Persona: string;
 	Tone: string;
 	PinnedOpen: boolean;
+	SuggestionsOpen: boolean;
```

Comment:
Not entirely sure we should be keeping state for these two. I'd think we'd want everything open every time the panel is being opened. I'm worried that users might fold some section and then think it doesn't work? What are your thoughts?

Reply:
Yeah, It's a fair point, and I'm actually inclined to agree with you, I think the user experience would be better with these expanded by default. I'm going to remove those settings.